### PR TITLE
Include network policies for Bedrock

### PR DESCRIPTION
**Pull request by ujjwalchk-it**
This PR is to include the Bedrock network policies .
https://github.ibm.com/IBMPrivateCloud/roadmap/issues/50359

### CODE REVIEW

File: networkpolicy/00-default-deny-all.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1
```
Comment (pgodowski) [inline networkpolicy/00-default-deny-all.yaml:1]: pls remove this file

File: networkpolicy/01-default-allow-all.yaml.bak [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: networking.k8s.io/v1
```
Comment (pgodowski) [inline networkpolicy/01-default-allow-all.yaml.bak:1]: pls remove this file

File: networkpolicy/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Overview
```
Comment (pgodowski) [inline networkpolicy/README.md:1]: Pls provide context and link to Knowledge Center

**Review by pgodowski — CHANGES_REQUESTED** [review]

File: networkpolicy/install_networkpolicy.sh [hunk 4]
```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+# Licensed Materials - Property of IBM
+# Copyright IBM Corporation 2021. All Rights Reserved
```
Comment (pgodowski) [inline networkpolicy/install_networkpolicy.sh]: Pls update copyright year

File: networkpolicy/README.md (hunk 3 above)
Comment (ujjwalchk-it) [reply networkpolicy/README.md:1]: Added

**Review by pgodowski — APPROVED** [review]

File: networkpolicy/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+-----------------------------------------------------------------------
+[✔] oc command available
+[✔] oc command logged in as [redacted]
+[✔] IBM Common Services found in namespace ibm-common-services
+# [0] Installing Bedrock Network Policies ...
+-----------------------------------------------------------------------
+ibm-common-services
+ibm-common-services
+[INFO] Installing 00-default-deny-all.yaml ...
+networkpolicy.networking.k8s.io/default-deny-ingress created
```
Comment (horis233) [inline networkpolicy/README.md:21]: @ujjwalchk-it Should we should the deny-all information from the Readme as well?

File: networkpolicy/README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+```
+networkpolicy % ./install_networkpolicy.sh -u
+# [0] Checking prerequesites ...
+-----------------------------------------------------------------------
+[✔] oc command available
+[✔] oc command logged in as [redacted]
+[✔] IBM Common Services found in namespace ibm-common-services
+# [0] Removing Bedrock Network Policies ...
+-----------------------------------------------------------------------
+networkpolicy.networking.k8s.io "default-deny-ingress" deleted
```
Comment (horis233) [inline networkpolicy/README.md:64]: Should we should deny-all information from the Readme as well?

**Review by horis233 — APPROVED** [review]

**Pull request merged**